ansible/roles_ocp_workloads/ocp4_workload_rhacm_clusters/tasks/cluster_kconf.yml

```diff
@@ -1,8 +1,10 @@
 ---
 - name: Set ACM cluster variables
   set_fact:
-    kconf_cluster_name: "{{ _ocp4_workload_rhacm_clusters_aws_clusters_item.managed_cluster_name }}"
-    cluster_kconf_path: "{{ _ocp4_workload_rhacm_clusters_aws_clusters_item.managed_cluster_kconf_path | default('~/.kube/') }}"
+    kconf_cluster_name:
+      "{{ _ocp4_workload_rhacm_clusters_aws_clusters_item.managed_cluster_name }}"
+    cluster_kconf_path:
+      "{{ _ocp4_workload_rhacm_clusters_aws_clusters_item.managed_cluster_kconf_path | default('~/.kube/') }}"
```

Comment:
If breaking to next line please use `>-` and drop quotes.

Reply:
In next PR